src/sagas/ui.js

```diff
@@ -1,12 +1,17 @@
 import {all, call, put, select, take, takeEvery} from 'redux-saga/effects';
 import {debounceFor} from 'redux-saga-debounce-effect';
-import {userDoneTyping as userDoneTypingAction} from '../actions/ui';
+import {
+  userDoneTyping as userDoneTypingAction,
+  currentFocusedSelectorChanged,
+} from '../actions/ui';
 import {getCurrentProject} from '../selectors';
 import {
   projectExportDisplayed,
   projectExportNotDisplayed,
 } from '../actions/clients';
 import {openWindowWithContent} from '../util';
+
+import {selectorAtCursor} from '../util/selectorAtCursor';
```

Comment:
Right now `postcss` isn’t part of the main (blocking) bundle, and it would be good for load time to keep it that way! Let’s import this asynchronously, like we [do with validations](https://github.com/popcodeorg/popcode/blob/master/src/sagas/errors.js#L16-L23).

Reply:
Looked at validations example but couldn't quite figure the right way to implement for selector at cursor. what should this look like? @outoftime